README.md

```diff
@@ -54,10 +54,11 @@ To **find the next item in reading order**, given a reading order item _current_
 1. Let _reading order items_ be the list of reading order items owned by _scope_, sorted in **reading order**.
 2. If _reading order items_ is empty, return null.
 3. If _direction_ is “forward”, then:
-   1. Let _previous_ be the reading order item that comes before _current_, in DOM tree order.
-   2. If _previous_ is null, return the first item in _reading order items_.
-   3. Otherwise, if _previous_ is the last item in _readinging order items_, return null.
-   4. Otherwise, return the item that comes after _previous_ in \_reading order items.
+   1. If _current_ is null, let _previous_ be the last reading order item, in DOM tree order.
+   2. Otherwise, let _previous_ be the reading order item that comes before _current_, in DOM tree order.
+   3. If _previous_ is null, return the first item in _reading order items_.
```

Comment:
You can't say `previous is null` because nothing sets it to null if it's first. So you have to say something like the old text, e.g. "If previous is the last item in ..., return ...".

Reply:
You are right I wasn't clear on how _previous_ gets set to null. _previous_ gets set to null if _current_ is the first reading order item in DOM tree order since there will be no item before _current_ (step 2). 

I changed so this case is the first condition.
"If _current_ is the reading order item from _reading order items_ that comes first in DOM tree order, return first item in _reading order items_."